config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: :show do
+    resources :dish_ingredients, only: :create
+  end
+
+  resources :chefs, only: :show do
+    resources :ingredients, only: :index
+  end
```

Comment:
Nice job with these routes. Looks good.

Reply:
Also, can consider nesting controllers. You could have created a chefs directory under you controllers, then in there, created an IngredientsController, since you have them reliant on each other. Not critical, just a suggestion.